GliaWidgets/SwiftUI/Components/SnackBarView/SerialQueue.swift

```diff
@@ -0,0 +1,56 @@
+import Foundation
+
+/// Represents queue with serial strategy. Not thread-safe.
```

Comment:
Could expand this description. What is the it helps to achieve it the view?

Reply:
updated - f529ff1c062e6c8cf12a429d21cb10dc1f01d838